cmd/csi-node-driver-registrar/main.go

```diff
@@ -119,6 +121,14 @@ func modeIsKubeletRegistrationProbe() bool {
 }
 
 func main() {
+	// this should be done early to prevent spawning too many Go threads
+	var maxprocsWarning error // log this after logger has been setup
```

Comment:
I can see the handling is short & small, still, should it be in all CSI sidecars and therefore in csi-lib-utils? Or is it just node-driver-registrar issue?

Reply:
Indeed, all CSI sidecars would probably benefit from this enhancement. I'll consider placing this somehow in csi-lib-utils.